### ISSUE TITLE
put together a multi-instance guide

### ISSUE THREAD

**Issue by neuroscr** [loki-project/loki-launcher#23]
clone repo
create ini file
var_path needs to be distinct per copy

**Comment by neuroscr** [comment]
start all
`systemctl start lokid@snodeX.service`

stop all
`systemctl stop lokid@snodeX.service`

restart all
`systemctl restart lokid@snodeX.service`

upgrade binaries (is there only one copy)

**Comment by jagerman** [comment]
We should not do this.

Loki-launcher is great at lowering the barrier to running a service node, but I am concerned that if we lower the barrier to running *multiple* service nodes on the same VPS we are going to degrade the network.  For those that can assess that they do have sufficient system performance and network bandwidth there are options available, but I think having the default easy option only support single instance setups out of the box is a feature, not a bug.

**Comment by neuroscr** [comment]
yes, already decided will not, this will be move into a personal project of mine
